developer/graphql-intro.md

```diff
@@ -0,0 +1,148 @@
+# Getting started with GraphQL
+
+## Install
+
+- Make sure you are on Reaction 1.11 and above.
+- Check your current Reaction version by running: `reaction -v`
+
+> Note: The GraphQL API is an experimental, alpha prototype. Do **not** use for production.
+
+## Your first query
```

Comment:
Need to add: Log in as an admin first via http://localhost:3000/ -- or else you'll see Unauthorized